lib/manager/poetry/extract.ts

```diff
@@ -88,12 +88,12 @@ function extractFromSection(
 
 function extractRegistries(pyprojectfile: PoetryFile): string[] {
   const sources = pyprojectfile.tool.poetry.source;
+  const registryUrls: string[] = ['https://pypi.org/pypi/'];
```

Comment:
Actually we prefer to handle defaults within datasource, not manager. Otherwise it's too hard to override the default registry URL in Renovate config.

Reply:
Do you think 0256ad072b52286d47583d6feb64a930abe615c3 is enough? Unless I'm missing something, I don't think I can easily use the default registry from datasource if I have other registries to add. Hence, I've done something similar to what [`pip_requirements`](https://github.com/renovatebot/renovate/blob/27fd0f9811d3142f1eb9ff5b839c85c23ca8c3c1/lib/manager/pip_requirements/extract.ts#L45) manager does.